src/collection/views/EditCollectionMetadata.tsx

```diff
@@ -35,7 +35,17 @@ const EditCollectionMetadata: FunctionComponent<EditCollectionMetadataProps> = (
 								<FormGroup label="Vakken" labelFor="subjectsId">
 									<TextInput value={''} id="subjectsId" />
 								</FormGroup>
-								<FormGroup label="Korte omschrijving" labelFor="shortDescriptionId">
+								<FormGroup
```

Comment:
@bertyhell ik zie nu pas dat dit mee in die branch zat, is dat iets dat al mee mag?

Reply:
nee, ik heb het eruit gehaald